src/DeviceInterfaceValues.tsx

```diff
@@ -565,13 +594,11 @@ const UnsetDataModal = ({
               isInvalid={!!errors}
             >
               <option value="">Select an endpoint</option>
```

Comment:
I would disable this option, the user shouldn't be able to select it.

```suggestion
              <option value="" disabled>Select an endpoint</option>
```